app/src/main/java/com/concordium/wallet/ui/account/accountdetails/AccountDetailsViewModel.kt

```diff
@@ -105,8 +105,8 @@ class AccountDetailsViewModel(application: Application) : AndroidViewModel(appli
     val showPadLockLiveData: LiveData<Boolean>
         get() = _showPadLockLiveData
 
-    private var _transferListLiveData = MutableLiveData<List<AdapterItem>>()
-    val transferListLiveData: LiveData<List<AdapterItem>>
+    private var _transferListLiveData = MutableLiveData<List<AdapterItem>?>()
+    val transferListLiveData: MutableLiveData<List<AdapterItem>?>
```

Comment:
Seems like boyscouting, which I generally think is a good idea, but for Legacy wallet - I think we should keep changes to a minimum. In essence, we're just changing some text, but the amount of changes is overwhelming due to library updates, refactoring, etc. It makes reviewing the task at hand harder.

I would encourage you to make a separate PR for improvements next time - and only do minimum work on Legacy wallet

Reply:
Because the project had no support for building on Apple Silicon, I had to update Gradle and other libraries, as a result of that a lot of stuff was flagged as errors and had to make a lot of changes to build the app. Nothing out of the necessary was touched.

Reply:
Thanks for the explaining. I am just confused as to why we need support for Apple Silicon. I mean, can't we build it as "we" have previously done with a few text edits?

Reply:
Ok fair enough. Another reason why we need CI pipeline for building asap, so IDE and tools used on dev machines don't require "code changes" like this. Thanks for addressing my concerns. I have no further comments.